app/v1/hateoas/HateoasLinks.scala

```diff
@@ -66,19 +69,25 @@ trait HateoasLinks {
     )
 
   // State benefits Hateoas
+  def addStateBenefits(appConfig: AppConfig, nino: String, taxYear: String): Link =
+    Link(
+      href = baseUri(appConfig, nino, taxYear),
+      method = POST,
+      rel = ADD_STATE_BENEFIT
+    )
 
-  def updateStateBenefits(appConfig: AppConfig, nino: String, taxYear: String, benefitId: String): Link =
+  def updateStateBenefit(appConfig: AppConfig, nino: String, taxYear: String, benefitId: String): Link =
```

Comment:
Why did you change this??

Reply:
the endpoint is called `Update State Benefit` not `Update State Benefit**s**`